client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -80,12 +80,12 @@ const EmailProvidersStackedComparison = ( {
 
 	const resolvedDomainName = domain ? domain.name : selectedDomainName;
 
-	if ( ! resolvedDomainName ) {
+	if ( ! domain ) {
 		return <></>;
 	}
```

Comment:
I'm not sure why we still need `resolvedDomainName` then, given that we will bail out if `domain` is not available.